library/src/main/java/com/novoda/downloadmanager/lib/Cursors.java

```diff
@@ -0,0 +1,23 @@
+package com.novoda.downloadmanager.lib;
+
+import android.database.Cursor;
+
+final class Cursors {
```

Comment:
I don't think this class is a wise move :x: its secretly a `CursorUtil` and we don't want that. Decorate the cursor if you don't think it's methods are good enough